packages/v4/patternfly-docs/content/design-guidelines/components/alert/alert.md

```diff
@@ -5,68 +5,67 @@ related: ['Alert', 'Alert group']
- 
-## Elements
-Toast alerts are always dismissible, but bordered inline alerts can be both dismissable and non-dismissible. All other elements are consistent between toast and bordered inline alerts.
+## Elements: 
+Elements vary depending on the variation of Alert. Toast alerts are always dismissible, but bordered inline alerts can be both dismissable and non-dismissible. All other elements are consistent between toast and bordered inline alerts.
 Plain inline alerts are never dismissable, and consist of only a status area and alert title.
  
 <img src="./img/alert-types.png" alt="Alert types" width="868" />
- 
-1. **Status area**: The status area indicates the severity of the alert with a specific icon and color.  
-2. **Alert title**: The alert title concisely communicates the alert’s main message. Its text color is coded to match the alert type. 
-3. **Close button**: The close button is used to dismiss the alert. It’s required for toast alerts and optional for inline alerts. 
-4. **Actions (optional)**: The actions area enables the user to take a specific action in relation to the alert. 1 or more actions can be placed inside of an alert. 
+
+1. **Status area**: The status area indicates the severity of the alert with a specific icon and color. 
+
+2. **Alert title**: The alert title concisely communicates the alert’s main message. Its text color is coded to match the alert type.
+
+3. **Close button**: The close button is used to dismiss the alert. It’s required for toast alerts and optional for inline alerts.
+
+4. **Actions (optional)**: The actions area enables the user to take a specific action in relation to the alert. 1 or more actions can be placed inside of an alert.
+
 5. **Description (optional)**: The description field communicates additional information to the user. It can also include a navigation link as an alternative to an action.
+
 6. **Expandable alert (optional)**: The expansion reveals additional information.
- 
-## Communicating severity
+### Communicating severity
 The status icon, background, and alert title are coded to communicate the severity of an alert.
- 
 <img src="./img/alert-status.png" alt="alert styles" width="700" />
- 
-|Type 	|Icon	|Usage  	|
+|Type   |Icon   |Usage      |
 |---------|--------|--------------|
 |Default  |fa-bell | Use for generic messages with no associated severity
-|Information 	|fa-info-circle|Use for general informational messages|
+|Information    |fa-info-circle|Use for general informational messages|
 |Warning  |fa-exclamation-triangle |Use to indicate that a non-critical error has occurred|
 |Critical |fa-exclamation-circle | Use to indicate that a critical or blocking error has occurred
 |Success  |fa-check-circle | Use to indicate that a task or process has completed successfully
- 
+
 For more information about what colors to use for status and severity, visit the [Colors page](/guidelines/colors/#status-and-state-colors).
  
-## Using alerts
+## Variations
```

Comment:
I wonder if we should restructure this section to be like

H3: Bordered inline alerts
-- using bordered inline alerts

H3: Plain inline alerts
-- using plain inline alerts

H3: Toast alerts
-- using toast alerts

H3: